gslib/tests/testcase/integration_testcase.py

```diff
@@ -939,6 +939,7 @@ def VerifyObjectCustomAttribute(self,
     self.assertEqual(expected_value, value)
 
   def VerifyPublicAccessPreventionValue(self, bucket_uri, value):
+    # TODO: Delete this method in favor of VerifyCommandGet
```

Comment:
The edits to this file seem WIP

Reply:
This will be removed once backend fix a bug on their end.